### ISSUE TITLE
Deploy Spring boot app to embedded Websphere Liberty Profile

### ISSUE THREAD

**Issue by sri420** [spring-projects/spring-boot#7976]
Hi,

I would like to use embedded Websphere Liberty Profile(WLP) for running my spring boot application similar to embedded tomcat.

Is there any support for that with Spring boot. If not are there any alternative options.

Thanks,
Sri

**Comment by wilkinsona** [comment]
Thanks for getting in touch, but it feels like this is a question that would be better suited to [Stack Overflow](http://stackoverflow.com/). As mentioned in [the guidelines for contributing](https://github.com/spring-projects/spring-boot/blob/master/CONTRIBUTING.adoc#using-github-issues), we prefer to use GitHub issues only for bugs and enhancements. Feel free to update this issue with a link to the re-posted question (so that other people can find it) or add some more details if you feel this is a genuine bug.

**Comment by sri420** [comment]
Thanks for your response. I have already posted a similar query in stackoverflow.
http://stackoverflow.com/questions/41629564/use-embedded-wlp-to-run-spring-boot-application

Since I did not see explicit support for WLP in Spring Boot as per the docs and we have a requirement to use WLP , wanted to know if it would be supported in near future. 

Also, am looking for suggestions from others who may have faced similar issues so that I can leverage on their expertise.
